src/components/Dashboard/Dashboard.module.scss

```diff
@@ -1,42 +1,56 @@
-$base-unit: 4px;
-$base-font-size: 0.3rem;
 $color-blue: #041187;
 $color-red: #e30062;
 $color-green: #85da6b;
-$color-disabled: #e5e7eb;
+$color-disabled: #7c7e82;
 $color-white: #fff;
 $color-black: #000;
-$diff-margin: $base-unit * 15;
-$similar-margin: $base-unit * 8;
-$section-padding-top-bottom: $base-unit * 30;
-$section-padding-left-right: $base-unit * 25;
 
 .container {
-    margin: $diff-margin auto;
-    width: 85%;
+    margin-top: 40px;
+    display: flex;
+    justify-content: center;
+    line-height: 2rem;
+    align-items: center;
 }
 
-.searchLabel {
-    font-size: $base-font-size * 5;
-    padding-right: $base-unit * 8;
-    color: $color-blue;
+.searchBar {
+    width: 600px;
+    font-family: sans-serif;
+    margin-right: 8px;
+    border-radius: 4px;
+    font-size: 1.2rem;
+    padding: 0.5rem;
 }
 
-.searchBar {
-    background-color: $color-disabled;
-    border-radius: $base-unit * 2;
+.btnActive {
+    background-color: $color-blue;
+    border-radius: 4px;
+    color: white;
+    background-color: $color-blue;
     border: none;
-    width: 80%;
-    height: $base-unit * 15;
-    font-size: $base-font-size * 5;
+    padding: 12px 60px;
+    cursor: pointer;
 }
 
-.searchBtn {
-    margin-left: $base-unit * 8;
+.btnDisabled {
     background-color: $color-blue;
-    color: $color-white;
-    height: $base-unit * 15;
-    border-radius: $base-unit * 2;
+    border-radius: 4px;
+    color: white;
+    background-color: $color-disabled;
     border: none;
-    width: $base-unit * 40;
+    padding: 12px 60px;
+    cursor: pointer;
+}
+
+@media only screen and (max-width: 520px) {
+    .container {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        align-items: center;
+    }
+    .searchBar {
+        width: 100%;
+        margin: 4px;
+    }
```

Comment:
Not a change but you could have directly set a `max-width: 600px` along with `width: 100%` so that the `max-width` would apply when the screen size is greater than `600px` and otherwise `width: 100%`. This way you wouldn't have to separately set a media query.

Reply:
Adding maximum width makes the component to be fixed to specified width in case it goes more than the mentioned width but the input tag here is way less than 600px so we add a width here

This is how it would look with max width on big screens
![image](https://github.com/Real-Dev-Squad/website-status/assets/57758447/fbe2b266-64a9-45a1-a0c3-9eca2277e65a)

Reply:
and this is how it would look if I put max width in media query:
![image](https://github.com/Real-Dev-Squad/website-status/assets/57758447/487fb860-b4c4-4f1c-9b80-541cb1466414)